doc/developers/testsuite.html

```diff
@@ -15,22 +15,23 @@
     <h1>The deal.II Testsuite</h1>
 
     <p>
-      deal.II has a testsuite that has, at the time this article was last
-      updated (April 2015), some 3,600 small programs (growing by roughly
-      one per day) that we run every time we make a change to make sure
-      that no existing functionality is broken. The expected output for
-      every test is stored in an <code>*.output</code>, and when you run a
-      test you are notified if a test produces different output.
+      deal.II has a testsuite with thousands of small programs that we run every time
+      we make a change to make sure
+      that no existing functionality is broken. It is located in the <code>tests/</code>
+      directory of the development repository. The expected output for
+      every test is stored in an <code>*.output</code> next to the <code>.cc</code> file,
+      and when running a test, you are notified if the output differs from the expected
+      output.
     </p>
 
     <p>
       These days, every time we add a significant piece of functionality,
       we add at least one new test to the testsuite, and we also do so if
       we fix a bug, in both cases to make sure that future changes do not
       break this functionality (again). Machines running the testsuite
-      submit the results to
+      submit the results to our cdash 
       <a href="http://cdash.kyomu.43-1.org/index.php?project=deal.II"
-      target="body">a webpage showing the status of our regression tests</a>.
+      target="body">CDash instance</a>, a webpage showing the status of our regression tests.
```

Comment:
i think if u drop `target="body"` it will open like a full page instead of being embedded into the previous.